### PR TITLE
Update sndfile.pc for MPEG support

### PR DESCRIPTION
**Pull request by evpobr**

### CODE REVIEW

File: configure.ac [hunk 1]
```diff
@@ -423,6 +421,7 @@ AS_IF([test -n "$PKG_CONFIG"], [
 				AS_IF([test "x$ac_cv_lame$ac_cv_mpg123" = "xyesyes"], [
 						enable_mpeg="yes"
 						HAVE_MPEG=1
+						EXTERNAL_MPEG_REQUIRE="libmpg123"
```
Comment (SoapGentoo) [inline configure.ac:424]: @evpobr had to fix the variable, and used `EXTERNAL_MPEG_REQUIRE` to be more consistent with the rest.

File: CMakeLists.txt [hunk 2]
```diff
@@ -219,6 +219,23 @@ if (INSTALL_PKGCONFIG_MODULE)
...
+		if (CMAKE_IMPORT_LIBRARY_PREFIX)
+			string (REGEX REPLACE "^${CMAKE_IMPORT_LIBRARY_PREFIX}" "" LAME_WE_NO_PREFIX ${LAME_WE})
+			if (LAME_HIP_LIBRARY)
+				string (REGEX REPLACE "^${CMAKE_IMPORT_LIBRARY_PREFIX}" "" LAME_HIP_WE_NO_PREFIX ${LAME_HIP_WE})
+			endif ()
+		endif ()
+		set (MPEG_LIBS "-l${LAME_WE_NO_PREFIX}")
+		if (LAME_HIP_LIBRARY)
+			set (MPEG_LIBS "${MPEG_LIBS} -l${LAME_HIP_WE}")
+		endif ()
```
Comment (SoapGentoo) [inline CMakeLists.txt:237]: I'm unhappy with `MPEG_LIBS`: If you drop `libmpg123` as a `Requires.private` into the `.pc`, that obviates the need for specifying `MPEG_LIBS` in `Libs.private`. I strongly suggest you get rid of the `Libs.private` line.

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

File: CMakeLists.txt (hunk 2 above)
Comment (evpobr) [reply CMakeLists.txt:237]: No, it is not libmpg123, it is LAME. It has no `.pc` file.

Comment (SoapGentoo) [reply CMakeLists.txt:237]: oh sorry, missed that, :+1:

**Comment by evpobr** [comment]
@SoapGentoo ?

**Review by SoapGentoo — APPROVED** [review]

**Pull request merged**